### PR TITLE
Final commit with all the files added , hopefullt didn't miss anything.

### LINKED ISSUE #41: Newsletter Signup Page
**Project title**
Newsletter Signup

**Project Description**
I want  to add a newsletter signup page which has api call with a newsletter.
@khushi-purwar If you find this to be required Please assign me this issue, I am a GSSOC'22 contributor.

**Stack**:  
- [x] Html
- [x] CSS
- [x] JavaScript
- [x] Node.js

### PR DESCRIPTION
**Pull request by uhini0201**
<hr>

## Description 📜

I have added my project which is a newsletter signup page, fully functional, deployed and responsive. 

Fixes #41 

<hr>

## Type of change 📝

Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this

- [x] Add a new project

<hr>

## Domain of Contribution 📊

Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this

- [x] Project using only HTML & CSS
- [x] Project using HTML, CSS & JS

<hr>
 
## Checklist ✅

Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this

- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GSSoC'22 contributor

<hr>
 
## Screenshots / Gif (Optional) 📸

Add the screenshot of the changes you have made

output1
output2
output3

### CODE REVIEW

**Comment by uhini0201** [comment]
@meinhoonharsh please review this and let me know the changes.

File: uhini_newsletter/app.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+app.post("/",function(req,res) {
+  var firstName = req.body.fname;
+  var lastName = req.body.lname;
+  var email = req.body.email;
+  var data = {
+    members :[
+      {
+        email_address: email,
+        status: "subscribed",
```
Comment (meinhoonharsh) [inline uhini_newsletter/app.js:20]: It's a Node Application, and it does not follow the Project Tech Stacks 
So I am unable to allow this to merge

**Review by meinhoonharsh — CHANGES_REQUESTED** [review]

**Comment by uhini0201** [comment]
@meinhoonharsh i have asked @khushi-purwar whether I can do it or not, she said I can as it is deployed one can see the final project too. Since she allowed it, i coded it.

**Comment by uhini0201** [comment]
@meinhoonharsh ????

**Comment by meinhoonharsh** [comment]
@khushi-purwar Please Look into it

**Comment by uhini0201** [comment]
@khushi-purwar can you please look into it??

**Comment by uhini0201** [comment]
@khushi-purwar Please update what else is needed to be done.
@meinhoonharsh

**Pull request closed without merging**